### DISCUSSION TITLE
cannot curl vaultwarden container from reverse proxy container

### DISCUSSION THREAD

**Discussion by DerTechn1ktyp** [dani-garcia/vaultwarden#5233] [Q&A] [answered]
Hey all,

I have a problem, which was not existing in the past. Unfortunately I was not able to figure it out where it come from.
Situation: I have a container called vaultwarden with rocket on port 8080 and a second container with swag, where my nginx with reverse proxy is running in.
Both containers are in the same docker network, so I'm able to resolve the containers by the internal DNS. Unfortunately I'm not able to reach the vaultwarden container on the rocket port on the docker network. The name of the container is successfully resolved to the container IP. The only way to reach the vaultwarden container is to use the IP address and the exposed port of the server where the containers are running at. Other containers are correctly reachable from the docker network (for example nextcloud). Do you guys have the same behavior?

`root@3a9538c22ea8:/#ping vaultwarden
PING vaultwarden (172.18.0.3): 56 data bytes
^C
--- vaultwarden ping statistics ---
2 packets transmitted, 0 packets received, 100% packet loss
root@3a9538c22ea8:/#curl http://vaultwarden:8080
curl: (7) Failed to connect to vaultwarden port 8080 after 195 ms: Could not connect to server
root@3a9538c22ea8:/#curl http://172.18.0.3:8080
curl: (7) Failed to connect to 172.18.0.3 port 8080 after 3052 ms: Could not connect to server
root@3a9538c22ea8:/#curl http://192.168.178.180:8080
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="color-scheme" content="light dark">
    <title>404 Not Found</title>
</head>
<body align="center">
    <div role="main" align="center">
        <h1>404: Not Found</h1>
        <p>The requested resource could not be found.</p>
        <hr />
    </div>
    <div role="contentinfo" align="center">
        <small>Rocket</small>
    </div>
</body>
</html>root@3a9538c22ea8:/#`

**Comment by BlackDex** [comment, marked as answer]
I do not run swag, but i have nginx running on a different container then Vaultwarden.
But that works just fine.

I know that docker sometimes messes things up after updates and that sometimes you either need to fully stop docker and start it again, or when using compose, sometimes `docker compose down ; docker compose up -d` works.

**Comment by DerTechn1ktyp** [comment]
Thank you so much! I did it a bit different, but you definitely pushed me into the correct direction. I just did a `sudo docker rm vaultwarden` and afterwards just created the container again with all the parameters. Now it's correctly reachable. Thank you so much! Much appreciated!
